build.gradle

```diff
@@ -85,6 +85,9 @@ allprojects {
             java.srcDir 'src/test/java/unstable'
             compileClasspath += main.output + test.output
             runtimeClasspath += main.output + test.output
+
+            ext.guiTestResources = "${sourceSets.test.resources.getSrcDirs()[0]}/guitests"
+            ext.unstableTestResources = "${sourceSets.test.output.resourcesDir}/unstable"
```

Comment:
yeah something like this, although i wonder if something like this would work?

```
sourceSets {
        unstableTests {
            ...
            output.resourcesDir = "${sourceSets.test.output.resourcesDir}"
        }
    }
```